src/entitites.json

```diff
@@ -0,0 +1,21 @@
+[
```

Comment:
make this a normal script file - JSON is awkward
also that allows you to use TS types

instead of a list, use a map - entity names have to be unique, and the map keys will generate a TS error if they collide

thus:

```
type EntityMap = {
  [name: string]: {
     fields: EntityField[]
  }
}
```

Reply:
Thanks for the feedback ! I made it a map in a ts file